pkg/agent/config/node_config.go

```diff
@@ -86,4 +99,5 @@ type NetworkConfig struct {
 	TunnelType        ovsconfig.TunnelType
 	EnableIPSecTunnel bool
 	IPSecPSK          string
+	DefaultMTU        int
```

Comment:
Maybe add a comment to state that this field is set to 0 if no value was provided by the user in the Agent's config.

Reply:
N/A